builtin/settingtypes.txt

```diff
@@ -366,6 +366,15 @@ enable_shaders (Shaders) bool true
 #    enhanced, highlights and shadows are gradually compressed.
 tone_mapping (Filmic tone mapping) bool false
 
+#    Adjust the saturation (or vividness) of the scene
+#    Values
+#    < 1.0 decrease saturation
+#    > 1.0 increase saturation
+#    1.0 = unchanged saturation
+#    0.0 = black and white
+#    (Tone mapping needs to be enabled.)
+saturation (saturation) float 1.0 0.0 5.0
```

Comment:
```suggestion
saturation (Saturation) float 1.0 0.0 5.0
```

By the way, is it fine to use just `saturation` (not `display_saturation` for example)?

Reply:
I think since it's part of tone mapping that seems to be implied.